### PR TITLE
ENH: Adding conversion from VNL objects to Numpy objects

### PR DESCRIPTION
**Pull request by fbudin69500**
Adding conversion:
* VNL vector to Numpy array
* Numpy array to VNL vector
* VNL matrix to Numpy array
* Numpy array to VNL matrix

### CODE REVIEW

**Comment by fbudin69500** [comment]
@thewtex

**Comment by thewtex** [comment]
CC: @aylward @mjallais

File: include/itkPyVnl.hxx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+template<class T>
+PyObject *
+PyVnl<T>
+::_GetArrayFromVnlVector( VectorType * vector)
+{
+  PyObject *                  memoryView    = NULL;
+  Py_buffer                   pyBuffer;
+  memset(&pyBuffer, 0, sizeof(Py_buffer));
+
+  size_t                      pixelSize     = sizeof(DataType);
```
Comment (thewtex) [inline include/itkPyVnl.hxx]: pixelSize -> elementSize ?

File: include/itkPyVnl.hxx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+#ifndef itkPyVnl_hxx
+#define itkPyVnl_hxx
+
+#include "itkPyVnl.h"
+#include <stdexcept>
+
+namespace itk
+{
+
+template<class T>
```
Comment (thewtex) [inline include/itkPyVnl.hxx]: T -> TElement ?

**Comment by aylward** [comment]
Outstanding!

**Comment by fbudin69500** [comment]
@thewtex Updated code following your recommendations. Thanks for the review!

**Pull request merged**